### ISSUE TITLE
Stuck

### ISSUE THREAD

**Issue by bjggi** [ffWiener/EschaWarp#1]
Hello

Great Addon, thx for ur work. Using it on multible Chars on the same Time //send @all ew z , almost always one Char gets stuck, he doesnt warp and i cant move the Char and i will have to //exit the Game. Reset didnt work.

Do you maybe know whats causing it? Or how i could prevent it?

Thanks

**Comment by ffWiener** [comment]
This is a common issue with sending the same commands on multiple characters at the same time.  I'll look into another command to issue a warp to all characters with some spaced out timing similar to the superwarp plugin.

In the meantime, people seem to put in the spaced out timing in a way of their own.

**Comment by bjggi** [comment]
Yeah, having the same Problem with Sammehs Homepoint addon. But wow i didnt know about the Superwarp addon, thx /

Looking forward to the fix, thank you
